### ISSUE TITLE
is site out of date?

### ISSUE THREAD

**Issue by CroMarmot** [Mrbunker/userjs-jop#1]
https://github.com/Mrbunker/userjs-jop/blob/f37890e59eca9df8dee7deca60d30d86ed3b109f/vite.config.ts#L7

```diff
- const connectList = siteList.map((site) => site.hostname).concat(["javdb008.com", "g64w.com"]); 
+ const connectList = siteList.map((site) => site.hostname).concat(["javdb368.com", "g64w.com"]); 
```

**Comment by Mrbunker** [comment]
thanks

**Comment by CroMarmot** [comment]
fixed at 476dd008cd0726ccec7dbb4ca2928a47b5af301f

released in 93a51282b6f63c092eaa07037ca6e6aabe83df59
